packages/foam-vscode/src/test/support/extended-vscode-environment.js

```diff
@@ -8,7 +9,13 @@ class ExtendedVscodeEnvironment extends VscodeEnvironment {
     // Implementation of getWordRangeAtPosition uses "instanceof RegExp" which returns false
     // due to Jest running tests in the different vm context.
     // See https://github.com/nodejs/node-v0.x-archive/issues/1277.
+    // And also https://github.com/microsoft/vscode-test/issues/37#issuecomment-700167820
     this.global.RegExp = RegExp;
+    this.global.vscode = vscode;
+  }
+  async teardown() {
+    this.global.vscode = {};
```

Comment:
probably it doesn't matter, but should we try to put back whatever was overridden at setup time, if we are going to assign it in any case?

Reply:
Happy to do it if you like, but tests seem fine with it. Let me know your preference

Reply:
just to keep things consistent and avoid random unexpected behaviors, it feels to me that we should restore whatever was here before